docker/docker-entrypoint.d/30-sed-on-appconfig.sh

```diff
@@ -93,6 +93,15 @@ echo "replace APP_CONFIG_ECM_HOST"
     -i "${NGINX_ENVSUBST_OUTPUT_DIR}/app.config.json"
 fi
 
+if [[ -n "${APP_CONFIG_PLUGIN_AOS}" ]]; then
```

Comment:
this code should not be any more needed since we are using subenv on ACA
When the docker image is build Nginx should replace the values as part of the tpl with the env variable values